pkg/operator/hive/hiveadmission.go

```diff
@@ -52,6 +56,14 @@ func (r *ReconcileHiveConfig) deployHiveAdmission(haLog log.FieldLogger, h *reso
 	if r.hiveImage != "" {
 		hiveAdmDeployment.Spec.Template.Spec.Containers[0].Image = r.hiveImage
 	}
+	if hiveAdmDeployment.Annotations == nil {
+		hiveAdmDeployment.Annotations = map[string]string{}
+	}
+	if hiveAdmDeployment.Spec.Template.ObjectMeta.Annotations == nil {
+		hiveAdmDeployment.Spec.Template.ObjectMeta.Annotations = map[string]string{}
+	}
+	hiveAdmDeployment.Annotations[aggregatorClientCAHashAnnotation] = instance.Status.AggregatorClientCAHash
+	hiveAdmDeployment.Spec.Template.ObjectMeta.Annotations[aggregatorClientCAHashAnnotation] = instance.Status.AggregatorClientCAHash
```

Comment:
Cool, I didn't think setting an annotation would do anything, I see now it's in the template. TIL.